test/test_dtype.py

```diff
@@ -63,7 +63,7 @@ class TestDType(unittest.TestCase):
   @classmethod
   def setUpClass(cls):
     if not cls.DTYPE or not is_dtype_supported(cls.DTYPE): raise unittest.SkipTest("dtype not supported")
-    cls.DATA = rand_for_dtype(cls.DTYPE, 10)
+    cls.DATA = rand_for_dtype(cls.DTYPE, 8)
```

Comment:
This is a global change to all dtype tests, is that what you intended?

Reply:
Reverted, have used DATA[:8] instead for the bitcast test